### ISSUE TITLE
Docs: Akka Persistence Example

### ISSUE THREAD

**Issue by sebastianharko** [akka/akka#23433]
Hi,
I think that the Akka Persistence page needs a "real example"... The current example ( with _ExamplePersistentActor_ and _ExampleState_ ) is too abstract and most people don't think like that .... I think we should replace ExamplePersistentActor with something more concrete (e.g., ShoppingCart - which is an over-used example though - not sure if it's a good or bad)  ...What do you think ?
Cheers!

**Comment by ktoso** [comment]
I'm not sure about "real example" 😉  Allow me to explain:

This is the reference documentation, and it's meant to be rater dry. A full real shopping cart example could be a full page of descriptions and techniques and things one may have to think about. So not sure the "real example" is what we want in the reference docs - it would absolutely fit the tech-hub and be a "persistence quickstart" though - that would be really awesome.

// We have build an akka quickstart, and are in the middle of an akka http quickstart https://github.com/akka/akka-http-scala-seed.g8/pulls so a persistence one would be awesome if you could help.

If this is just about changing the ExampleActor to be a simple ShoppingCart example then this is fine, though the "real" word made me worry you had the "complete full real example" in mind :)

**Comment by sebastianharko** [comment]
Yes!  I meant just changing ExampleActor to be a simple ShoppingCart :-) 

We already have a less abstract example in the same page actually (the example for Persistent FSM is not abstract but concrete ... (see: http://doc.akka.io/docs/akka/snapshot/scala/persistence.html#a-simple-example) :-)

**Comment by johanandren** [comment]
The distributed workers quickstart will also feature an actual use case for persistence.

But I agree, it would be nice if that sample was less abstract. Preferably following on the lines of the getting started guide, so some kind of IOT/sensor thing to tie the docs together.

**Comment by patriknw** [comment]
I'm If we can make a somewhat more real example that still illustrates the API without to much domain boilerplate I'm all for it. Would you like to give it a try @sebastianharko ? Shopping cart domain is fine.

**Comment by sebastianharko** [comment]
@patriknw yes - I'll do the shopping cart domain and keep it very simple. Thanks !
